core/workflow/loadDPL.go

```diff
@@ -38,25 +36,20 @@ func LoadDPL(tasks []*task.Class) (workflow Role, err error) {
 		SingleTaskRole := taskRole{
 			roleBase: roleBase{
 				Name:        taskItem.Identifier.Name,
-				parent:      root,
-				Connect:     taskItem.Connect,
-				Constraints: taskItem.Constraints,
-				status: SafeStatus{
-					mu:     sync.RWMutex{},
-					status: 0,
-				},
-				state: SafeState{
-					mu:    sync.RWMutex{},
-					state: 0,
-				},
-				Defaults: gera.MakeStringMapWithMap(taskItem.Defaults.Raw()),
-				Vars:     &gera.StringWrapMap{},
-				UserVars: &gera.StringWrapMap{},
-				Locals:   nil,
-				Bind:     taskItem.Bind,
+				Connect:     nil,
+				Constraints: nil,
+				Defaults:    gera.MakeStringMap(),
+				Vars:        gera.MakeStringMap(),
+				Bind:        nil,
```

Comment:
Just to clarify: the question that resulted in this change was which fields should not appear in the template file, not which should be left empty. The difference is subtle but could be important.
Specifically `parent` is something you probably want to still set to `root`, even though it doesn't end up in the WFT, because when generating channel paths you might need to be able to navigate the workflow tree (`GetParent()`/`GetPath()` functions and related).